src/test/kotlin/apply/ui/api/EvaluationRestControllerTest.kt

```diff
@@ -0,0 +1,130 @@
+package apply.ui.api
+
+import apply.application.ApplicantService
+import apply.application.EvaluationData
+import apply.application.EvaluationResponse
+import apply.application.EvaluationService
+import apply.config.RestDocsConfiguration
+import apply.createEvaluation
+import apply.createRecruitment
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.every
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.FilterType
+import org.springframework.context.annotation.Import
+import org.springframework.http.MediaType
+import org.springframework.restdocs.RestDocumentationContextProvider
+import org.springframework.restdocs.RestDocumentationExtension
+import org.springframework.restdocs.mockmvc.MockMvcRestDocumentation
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.delete
+import org.springframework.test.web.servlet.get
+import org.springframework.test.web.servlet.post
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers
+import org.springframework.test.web.servlet.setup.DefaultMockMvcBuilder
+import org.springframework.test.web.servlet.setup.MockMvcBuilders
+import org.springframework.web.context.WebApplicationContext
+import org.springframework.web.filter.CharacterEncodingFilter
+import support.test.TestEnvironment
+
+@WebMvcTest(
+    controllers = [EvaluationRestController::class],
+    includeFilters = [
+        ComponentScan.Filter(type = FilterType.REGEX, pattern = ["apply.security.*"]),
+        ComponentScan.Filter(type = FilterType.REGEX, pattern = ["apply.config.*"])
+    ]
+)
+@Import(RestDocsConfiguration::class)
+@ExtendWith(RestDocumentationExtension::class, SpringExtension::class)
```

Comment:
```suggestion
    includeFilters = [
        ComponentScan.Filter(type = FilterType.REGEX, pattern = ["apply.security.*"]),
    ]
)
@Import(RestDocsConfiguration::class)
@ExtendWith(RestDocumentationExtension::class)
```

SpringExtension을 붙인 이유가 있을까요?

Reply:
엇 불필요한 코드였네요 ㅎㅎㅎㅎ
제거했습니다 감사합니다!!